go/vt/vtgate/planbuilder/testdata/foreignkey_cases.json

```diff
@@ -732,5 +807,255 @@
     "comment": "delete in a table with limit - disallowed",
     "query": "delete from u_tbl2 limit 2",
     "plan": "VT12001: unsupported: foreign keys management at vitess with limit"
+  },
+  {
+    "comment": "update with fk on cross-shard with a where condition on non-literal value - disallowed",
+    "query": "update tbl3 set coly = colx + 10 where coly = 10",
+    "plan": "VT12001: unsupported: foreign keys management at vitess with non-literal values"
```

Comment:
I think we can come up with a better error message for this limitation. this one is a little hard to decipher, imho